### PR TITLE
[Fix/#136] 기타 최종 수정

### PR DESCRIPTION
**Pull request by SebellKo**
## 📌 작업 내용

- 소상공인 리뷰 남긴 아이템 버튼 비활성화 처리
- 리뷰 남기기 수정

<br/>

## 📌 구현 결과 (선택)

<br/>

## 📌 기타 사항

감사합니다.

<br/>

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">7.86%</td> <td align="right">618 / 7860</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">7.86%</td> <td align="right">618 / 7860</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">15.8%</td> <td align="right">52 / 329</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">25.14%</td> <td align="right">89 / 354</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx">src/features/SideMenu/components/GroupBuy/SearchProduct.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L6-L9" class="text-red">6-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L13-L18" class="text-red">13-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L20" class="text-red">20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L22-L26" class="text-red">22-26</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L28-L29" class="text-red">28-29</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L31-L32" class="text-red">31-32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L34" class="text-red">34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L36-L39" class="text-red">36-39</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L41-L43" class="text-red">41-43</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L45-L49" class="text-red">45-49</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L51-L71" class="text-red">51-71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L73" class="text-red">73</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L75-L76" class="text-red">75-76</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L78" class="text-red">78</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L81" class="text-red">81</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L83" class="text-red">83</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L85" class="text-red">85</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L89" class="text-red">89</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L101" class="text-red">101</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L113-L114" class="text-red">113-114</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L117" class="text-red">117</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L120-L121" class="text-red">120-121</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L124" class="text-red">124</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L128" class="text-red">128</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/GroupBuy/SearchProduct.tsx#L138" class="text-red">138</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx">src/features/SideMenu/components/Mypage/ContentListItem.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L5" class="text-red">5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L7-L9" class="text-red">7-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L11-L13" class="text-red">11-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L15-L22" class="text-red">15-22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L24-L26" class="text-red">24-26</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L28-L56" class="text-red">28-56</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L58-L61" class="text-red">58-61</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L64-L65" class="text-red">64-65</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L67" class="text-red">67</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L69" class="text-red">69</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L71" class="text-red">71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L80-L82" class="text-red">80-82</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L85" class="text-red">85</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L87" class="text-red">87</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L90" class="text-red">90</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L103" class="text-red">103</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L109" class="text-red">109</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L113-L114" class="text-red">113-114</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L119" class="text-red">119</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L123-L125" class="text-red">123-125</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L128-L130" class="text-red">128-130</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L133-L135" class="text-red">133-135</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Mypage/ContentListItem.tsx#L138" class="text-red">138</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx">src/features/SideMenu/components/Review/ReviewForm.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L1-L12" class="text-red">1-12</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L19-L24" class="text-red">19-24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L26-L30" class="text-red">26-30</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L32-L34" class="text-red">32-34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L36-L46" class="text-red">36-46</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L48-L81" class="text-red">48-81</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L83-L98" class="text-red">83-98</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L100-L103" class="text-red">100-103</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L105" class="text-red">105</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L107" class="text-red">107</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L113" class="text-red">113</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L119" class="text-red">119</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L126" class="text-red">126</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L129" class="text-red">129</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L133-L135" class="text-red">133-135</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L138" class="text-red">138</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L142" class="text-red">142</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L149" class="text-red">149</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L155-L157" class="text-red">155-157</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L160" class="text-red">160</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L170-L172" class="text-red">170-172</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8e0e3ebb2dc64f697db76731e8d62bb4f78028ba/src/features/SideMenu/components/Review/ReviewForm.tsx#L175" class="text-red">175</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10644283269>#255</a></em>

File: src/features/SideMenu/components/Mypage/ContentListItem.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ function ContentListItem({ ...props }: GroupBuyListRes) {
 
   const ListItemData = { ...props };
   const ListType = ListItemData.status;
+  console.log(ListItemData.isWroteReview);
```
Comment (JIMIN1020) [inline src/features/SideMenu/components/Mypage/ContentListItem.tsx:13]: 삭제!

**Review by JIMIN1020 — APPROVED** [review]
세종님 끝까지 고생 많으셨습니다~~

**Review by juhee067 — APPROVED** [review]
수고많으셨습니다 세종사마~

**Pull request merged**